Description: Use the debian prototype package
 Use the Debian prototype package in place of the included version.
Author: Benoit Mortier <benoit.mortier@opensides.be>
Last-Update: 2012-01-26
Index: pkg/ihtml/themes/default/password.tpl
===================================================================
--- pkg.orig/ihtml/themes/default/password.tpl  2012-01-26 15:56:54.000000000 +0100
+++ pkg/ihtml/themes/default/password.tpl 2012-01-26 16:03:58.000000000 +0100
@@ -20,7 +20,7 @@
   <style type="text/css">@import url('themes/default/password-style.css');</style>
   <link rel="shortcut icon" href="favicon.ico">

-  <script language="javascript" src="include/prototype.js" type="text/javascript"></script>
+  <script language="javascript" src="/javascript/prototype/prototype.js" type="text/javascript"></script>
   <script language="javascript" src="include/fusiondirectory.js" type="text/javascript"></script>
   <script language="javascript" src="include/pwdStrength.js" type="text/javascript"></script>
 </head>
